compiler/rustc_span/src/hygiene.rs

```diff
@@ -1163,6 +1163,9 @@ pub enum DesugaringKind {
     WhileLoop,
     /// `async Fn()` bound modifier
     BoundModifier,
+    /// Marks a `&raw const *_1` needed as part of getting the length of a mutable
+    /// slice for the bounds check, so that MIRI's retag handling can recognize it.
```

Comment:
Also needs updating if Miri uses the `RawRefKind` rather than the span info

Reply:
See above